src/Common/ZooKeeper/ZooKeeperImpl.cpp

```diff
@@ -63,6 +63,39 @@ namespace CurrentMetrics
     extern const Metric ZooKeeperWatch;
 }
 
+namespace Metrics::ResponseTime
+{
+    Histogram::MetricFamily & mf = Histogram::Factory::instance().registerMetric(
+        "zookeeper_response_time_ms",
```

Comment:
We can use `"keeper_response_time_ms"` instead

Reply:
done